src/Libraries/Microsoft.AspNetCore.Telemetry.Middleware/Logging/Internal/HttpLoggingEnrichmentInterceptor.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NET8_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpLogging;
+using Microsoft.Extensions.Telemetry.Logging;
+
+namespace Microsoft.AspNetCore.Telemetry.Http.Logging;
+
+internal sealed class HttpLoggingEnrichmentInterceptor : IHttpLoggingInterceptor
+{
+    private readonly IHttpLogEnricher[] _enrichers;
+
+    public HttpLoggingEnrichmentInterceptor(IEnumerable<IHttpLogEnricher> httpLogEnrichers)
+    {
+        _enrichers = httpLogEnrichers.ToArray();
```

Comment:
I would log a warning/info here if there's no enrichers. It would mean that there's no point of calling `AddHttpLoggingEnrichment()`

Reply:
I'll just make it throw.

Reply:
I don't think that we need to fail the HTTP request because of a one failed logging enricher.
@CarnaViire suggested the same in HttpClient logging component.